Traceback (most recent call last):
  File "interface.py", line 45, in <module>
    interface()
  File "/dartfs-hpc/rc/home/p/f002z9p/.conda/envs/remmel/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/dartfs-hpc/rc/home/p/f002z9p/.conda/envs/remmel/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/dartfs-hpc/rc/home/p/f002z9p/.conda/envs/remmel/lib/python3.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/dartfs-hpc/rc/home/p/f002z9p/.conda/envs/remmel/lib/python3.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "interface.py", line 30, in interface
    pipe.select_features_cv()
  File "/dartfs-hpc/rc/home/p/f002z9p/robust-sffs-cluster/src/model.py", line 249, in select_features_cv
    get_parsimonious_k()
NameError: name 'get_parsimonious_k' is not defined
